### PR TITLE
cades:0.2.0

### PR DESCRIPTION
**Pull request by Midbin**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Cades is very simple package to generate qr codes in typst. This is, for example, useful when creating a presentation, or wanting to add a link to a printed document.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

**Comment by laurmaedje** [comment]
Where is the JavaScript code from / what license is it under?

**Comment by Midbin** [comment]
As stated in the readme the javascript "qr code rendering code is based on [qrcode-svg](https://github.com/papnkukn/qrcode-svg/) by [papnkukn](https://github.com/papnkukn)." That is also under the MIT license. I just combine it with a small helper function to call into the javascript code: [https://github.com/Midbin/cades](https://github.com/Midbin/cades).

**Comment by laurmaedje** [comment]
We probably still need some direct attribution, see discussion here: https://github.com/typst/packages/pull/228#issuecomment-1805364147

**Comment by Enter-tainer** [comment]
https://github.com/vitejs/vite/discussions/7722#discussioncomment-4007436 I haven't try it but it might be helpful

**Comment by Midbin** [comment]
I added a NOTICE which includes all the third party packages and their respective license (only qrcode-svg).

File: packages/preview/cades/0.2.0/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# [Cades](https://github.com/Midbin/cades)
+
+Draw QR codes in typst.
+
+```typ
+#import "@preview/cades:0.1.0": qr-code
```
Comment (laurmaedje) [inline packages/preview/cades/0.2.0/README.md:6]: There seems to be a mismatch between the versions here. 0.1.0 here, but 0.2.0 in the PR.

File: packages/preview/cades/0.2.0/typst.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+repository = "https://github.com/Midbin/cades"
+keywords = [
+    "qr",
+    "qr-code",
+    "qrcode",
+    "code",
+    "generate",
+    "generator",
+]
+exclude = []
```
Comment (laurmaedje) [inline packages/preview/cades/0.2.0/typst.toml:19]: An empty exclusion can be omitted.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**